### DISCUSSION TITLE
Doing pull export only when sync finished?

### DISCUSSION THREAD

**Discussion by rkrug** [retorquere/zotero-better-bibtex#2589] [General]
Probably already implemented, but is there a way of only starting the pull export, when the sync of the Group to be exported is finished?

Thanks.

**Comment by retorquere** [comment]
You mean that the pull export is requested and that I stall sending the output until Zotero sync is done?

**Comment by rkrug** [comment]
Yes - so that the export contains a consistent snapshot of the group library (I think it is only relevant for groups, when automatic sync is enabled?).

Actually - reading https://www.zotero.org/support/sync again, does Zotero automatically keep the local library in sync with the remote library (potentially edited by others) when simply left on auto sync?

**Reply by retorquere** [reply]
Yes. Not only relevant to groups, the main library is also synced.

**Reply by rkrug** [reply]
Good to know!

So the question is, is there a way to either stall or return a message when doing a pull export when the sync is still ongoing?

**Reply by retorquere** [reply]
Yes, but you'll have to open an issue for it.

**Comment by rkrug** [comment]
Ok. Will do so.
